pkg/types/ovirt/machinepool.go

```diff
@@ -29,6 +29,17 @@ type MachinePool struct {
 	// The affinity groups should exist on the oVirt cluster or created by the OpenShift installer.
 	// +optional
 	AffinityGroupsNames []string `json:"affinityGroupsNames"`
+
+	// AutoPinningPolicy defines the policy to automatically set the CPU
+	// and NUMA including pinning to the host for the instance.
+	// +kubebuilder:validation:Enum="";disabled;adjust
+	// +optional
+	AutoPinningPolicy AutoPinningPolicy `json:"autoPinningPolicy,omitempty"`
```

Comment:
It is still very unclear to me why we do not want to document what the behavior is when this field is omitted. The terraform code is explicitly using "disabled" in this case. I would expect there to be an install-config default established that sets this field to "disabled" when the user omits the field.